SINK.md

```diff
@@ -0,0 +1,39 @@
+# Sink Ruter
+
+This contains transaction data for the old 'Troms Billett' app
+develped by Ruter as part of the now dead KK 1.0 collaboration.
+
+The transaction in this app stood for the majority of our
+ticketing/product income in the period it was live.
+
+## Data
+
+The data set primarly consist of purchases from the old 'Troms
+Billett' app (terminated in september, 2024) and with it all possible
+meta data related to phone, app, product and its transactions. This is
+located in the `ruter_transactions` table.
+
+As it is possible to buy several tickets in one transaction, an
+additional table `ruter_passengers` contains information about all
+passengers and the product they has attached to the given purchase.
+
+Historically this data set has been unstable and prone to
+changes/errors and unannounced updates.
+
+Another thing to notice is the user identifier. There is no such. The
+only way to track users is by app instance name, which changes every
+time the app is installed.
```

Comment:
How about when the app was updated? We should document that

Reply:
It's sort of covered by
> and with it all possible metadata related to phone, app …

Reply:
But fixed with an additional 'app, app version, …'